src/components/ContentWidgets/Homepage/HomepageEditor.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FormEvent, ChangeEvent } from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import './Homepage.css';
+
+export const HomepageEditor: React.FC<WidgetEditorProps> = ({
+    originalContent, 
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let homepage_content = editedContent.homepage_content || 
+        {title: "Title not set", subtitle: "Subtitle not set", backgroundImage: "", blur: "", bgLightness: "", bgOpacity: ""};
+
+    const updateTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.title = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+
+    const updateSubtitle = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.subtitle = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+
+    const updateBackgroundImage = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.backgroundImage = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+
+    const updateBlur = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.blur = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+
+    const updateBgLightness = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.bgLightness = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+
+    const updateBgOpacity = (e: React.ChangeEvent<HTMLInputElement>) => {
+        homepage_content.bgOpacity = e.target.value;
+        setEditedContentOnChange("homepage_content", homepage_content);
+    }
+    
+    return <>
+        <div style={{
```

Comment:
You are free to actually just call the view component here. That would be more scalable so when we update the view component we dont have to update the editor component's view too.